src/traces/scattergl/attributes.js

```diff
@@ -82,6 +83,10 @@ var attrs = module.exports = overrideAll({
         marker: scatterAttrs.unselected.marker
     },
 
+    opacity: extendFlat({}, plotAttrs.opacity, {
+        editType: 'calc'
+    }),
```

Comment:
Looks good. Thanks!

I think the best way to test this would be to spy on `ScatterGl.calc` checking that `Plotly.relayout(gd, 'opacity', /**/)` does invoke it - similar to this [suite](https://github.com/plotly/plotly.js/blob/18dac1f4210378945baacbffdacc06f883fc55a7/test/jasmine/tests/plot_api_test.js#L569).